espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/MobileSwipe.kt

```diff
@@ -68,4 +76,20 @@ class MobileSwipe : RequestHandler<MobileSwipeParams, Void?> {
 
         return null
     }
+
+    override fun handleCompose(params: MobileSwipeParams): Void? {
+        // Get a reference to the compose node
+        val nodeInteractions = getNodeInteractionById(params.elementId)
+
+        require((param.direction !=null){"Must provide direction to swipe to :up,down,right,left"}
```

Comment:
Must provide direction to swipe to: up, down, right, left

Reply:
I think you could generate available direction names automatically from enum items

Reply:
Please format the line